Packaging/Package.appxmanifest

```diff
@@ -301,6 +301,45 @@
         </uap:Extension>
       </Extensions>
     </Application>
+    <Application Id="DevHome.HostsFileEditor" Executable="DevHome.HostsFileEditor\DevHome.HostsFileEditor.exe" EntryPoint="Windows.FullTrustApplication">

```

Comment:
Multi-App-Packages are rare, and this will surely cause some unexpected difficulties in the publishing or build pipeline. @EricJohnson327 should weigh in on this as it probably affects our Store publishing.

Also a general question of whether piling more into Dev Home is the correct thing to do, or whether we should have separate app packages for this like we have for the GitHub and Azure extensions. Seems like Dev Home is becoming very bloated very quickly and the trend is growing.

Reply:
[Spec](https://github.com/microsoft/devhome/blob/main/docs/specs/%232072%20-%20Add%20some%20PowerToys%20utilities/spec.md) : We want to deliver these PowerToys with the DevHome itself and hence this route of not creating separate packages.